metadata-ingestion-examples/common/src/main/java/com/linkedin/metadata/examples/configs/KafkaConfig.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.metadata.examples.configs;
```

Comment:
Wouldn't you want this under `metadata-ingestion-examples/kafka-etl` instead?

Reply:
No. When we port the "mce cli" (opportunity to rename there, but we can discuss in that PR...) it will reuse this.